pkg/oc/cli/cmd/startbuild.go

```diff
@@ -462,6 +454,29 @@ func (o *StartBuildOptions) Run() error {
 	return nil
 }
 
+func (o *StartBuildOptions) streamBuildLogs(build *buildapi.Build) error {
+	opts := buildapi.BuildLogOptions{
+		Follow: true,
+		NoWait: false,
+	}
+	var err error
+	for i := 0; i < streamLogAttempts; i++ {
+		rd, logErr := o.BuildLogClient.Logs(build.Name, opts).Stream()
+		if logErr != nil {
+			err = ocerrors.NewError("error getting logs").WithCause(logErr)
+			// retry after waiting a fixed period of time
+			time.Sleep(streamLogWaitDuration)
```

Comment:
i don't think we want to wait here.  the log api already waits 10s for the build to start running(right?), so i'd say just call it again immediately and let it do the waiting.